lib/Controller/SignalingController.php

```diff
@@ -118,10 +118,13 @@ public function __construct(string $appName,
 	/**
 	 * @PublicPage
 	 *
+	 * @param string $apiVersion
 	 * @param string $token
 	 * @return DataResponse
 	 */
-	public function getSettings(string $token = ''): DataResponse {
+	public function getSettings(string $apiVersion, string $token = ''): DataResponse {
+		$apiV = (int) substr($apiVersion, 1);
```

Comment:
I guess we don't worry that this might start breaking once we reach v10 :-D

Reply:
substr(…, 1) removes the first char (the `v`) not takes the last one. So it's #futureProof